cmd/minishift/cmd/root.go

```diff
@@ -177,22 +167,30 @@ func processEnvVariables() {
 func init() {
 	processEnvVariables()
 	RootCmd.PersistentFlags().Bool(showLibmachineLogs, false, "Show logs from libmachine.")
+	RootCmd.PersistentFlags().String(profile, "minishift", "Profile name")
```

Comment:
why use a constant?

I believe this even clashed with https://github.com/minishift/minishift/pull/1304/files#diff-caeade0d74523273373d2689861f618eR189 when not carefully used.

Reply:
Nope. That the default value. When user provides --profile PROFILE_NAME it gets overwritten with user provided value.

Reply:
And when user does not provide --profile then the value is null string.